.circleci/config.yml

```diff
@@ -149,7 +149,7 @@ workflows:
           matrix:
             alias: test-getting-started-guides-18
             parameters:
-              language: ["go", "java", "node-js", "typescript", "php", "python", "ruby"]
+              language: ["go", "java", "node-js", "typescript", "php", "python", "ruby", "gradle"]
```

Comment:
These were in alphabetical order, but fine with fixing that (and making the order consistent with that used for `heroku/builder-classic:22`'s test `matrix`) in the later PR to add `scala` to save another RTT

Reply:
👍🏻